.gitlab-ci.yml

```diff
@@ -8,6 +8,11 @@ include:
     ref: master
     file: /gitlab_ci_templates/jedi-gitlab-ci-template.yml
 
+.retrieve_source:
```

Comment:
I added this to the build step.

Reply:
It still fails, but I'm not going to tackle this head on today as we can still build locally and tag the built package in git. I'll make a separate ticket to fix um-kernel CI, as this doesn't block the building of the complete 7.0.3 firmware.

Reply:
Bummer. I'll pick this up next monday!